README.md

```diff
@@ -1,57 +1,125 @@
-# HQ9
+# HQ9+
+
 <!-- TOC depthFrom:2 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->
 
 - [Build](#build)
 - [Background](#background)
-  - [HQ9](#hq9-1)
+- [Implementation](#implementation)
   - [C](#c)
-    - [89](#89)
-  - [C89.5](#c895)
+    - [C89](#c89)
+      - [C89.5](#c895)
+  - [C++](#c-1)
+    - [C++20](#c20)
+- [Caveats](#caveats)
```

Comment:
Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/lucaslarson/HQ9/pullRequest?prid=6019912)

Reply:
✅ resolved https://github.com/LucasLarson/HQ9/pull/5/commits/fbc00b6a52587879e39a5c07db6dc90b138df801 (and https://github.com/LucasLarson/HQ9/pull/10#issuecomment-675084128)